<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(129, 61, 156, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 25pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>PYQT5 Design</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(246, 245, 244);
font: 15pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Ingresar Usuario Por Favor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(246, 245, 244);
font: 15pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(246, 245, 244);
font: 15pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Contrase√±a:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px; </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px; </string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="toolTip">
    <string extracomment="Dar click"/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(143, 240, 164);
border-radius: 10px; </string>
   </property>
   <property name="text">
    <string>Ingresar</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>460</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="toolTip">
    <string extracomment="Dar click"/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 97, 81);
border-radius:10px; </string>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 10pt &quot;Fira Mono&quot;;
color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
